apps/extension/src/routes/popup/approval/transaction.tsx

```diff
@@ -16,10 +26,10 @@ export const TransactionApproval = () => {
           Confirm transaction
         </p>
 
-        <TransactionViewComponent txv={TransactionView.fromJson(transactionViewFromPlan)} />
+        <TransactionViewComponent txv={transactionViewFromPlan} />
 
         <div className='mt-8'>
-          <JsonViewer jsonObj={authorizeRequest} />
+          <JsonViewer jsonObj={authorizeRequest.toJson() as Jsonified<AuthorizeRequest>} />
```

Comment:
I wonder if JsonViewer should take `JsonValue` (?)

Reply:
considered making that change but chose to limit scope

Reply:
ok, now JsonViewer wants JsonObject (we will probably never just hand it a bare primitive)